ChangeLog

```diff
@@ -1,3 +1,45 @@
+3.26.0:
+	- Fixed `cf-net connect` to return exit code 1 in case of error (CFE-4414)
+	- Added feature class for libpam (CFE-3371)
+	- Added hardclasses for log levels notice, warning and error (CFE-4126)
+	- Added policy variable sys.cpusockets (CFE-30)
+	- Added a new policy function to get the list of metatags of a bundle, in the same manner as getclassmetatags and getvariablemetatags.
```

Comment:
the changelog generator in misc wraps at 70 characters I think, so I will adjust accordingly.
```
            if len(entry) - entry.rfind("\n") + len(tracker) >= 70:
```